docs/Test-Strategy-Info.mdx

```diff
@@ -1,31 +1,168 @@
+These tests run the actual UI in a browser, and can check that everything renders correctly. For example, an integration test might check that “most read” topics are rendered with the correct HTML tags, but that does not necessarily mean they appear correctly (some CSS value might hide them). With the UI tests, we check that everything shows as expected in the UI.
+Examples and instructions can be found in [cypress](https://github.com/bbc/simorgh/tree/latest/cypress).
+
+
+
+> 🔊 discussion point with team: should we break UI / e2e tests as @Simon F comment mentions - I agree with the comment, but I am not sure this reflects what we do in our team right now?
+
+
+#### What levels to prioritise?
+
+Unit tests responsibility and scope are relatively easy to define: we already have a high level of code coverage and developers should aim to maintain that. 
+
+For integration and e2e tests, the general rule of thumb should be to only add an e2e test when an integration test is not enough to ensure coverage. Integration tests are less expensive and more stable and they should be prioritised.
+
+
+> ℹ️ Think about the maintenance cost of adding a new e2e test. Always favour an integration test if possible instead of an end to end test.
+
+For new features, we should decide as part of the planning or shaping of work/tickets whether an e2e test is necessary for the feature.
+
+### What should we automate?
+
+The decision to write an e2e test should be taken during planning as a team, but these are general guidelines to what should go into e2e tests
+
+#### Critical paths 
+The critical path is the set of components and workflows that is required for the application to serve its core function. For example, an article page displaying the content and title for the given service is critical in our context, but links to related topics or top stories, can be considered non-critical. These should be tested as well, but they’re better candidates for a less expensive layer of testing like integration tests.
+
+#### Smoke tests
+We should have smoke tests for validating that core functionality is not broken, for example, that the page loads, has titles and content. These should run as often as possible (on each PR), they should be quick, reliable and repeatable (not flaky).
+
+#### As little as possible
+The rule of thumb is that we should avoid writing an end to end test if there is an alternative to validating the functionality. The alternative can be a combination of layers, for example, an integration test combined with an API test (which doesn’t exist right now) can be enough to reach the same confidence as an e2e test with less cost.
+
+### Puma approach
+Adopting the [PUMA](https://confluence.dev.bbc.co.uk/display/podtest/PUMA) approach can provide a framework to decide if a proposed test genuinely meets the definition of an E2E. According to PUMA, an e2e test should:
+P - Prove core functionality, 
+U - be Understood by all,
+M - Mandatory
+A - Automated
```

Comment:
We had to add extra lines to make them appear as we want - this is what it looks like: ![Screenshot 2024-11-28 at 12 49 24](https://github.com/user-attachments/assets/7d66de41-0e64-4a63-8966-7ea97a6f96a1)

```suggestion
P - Prove core functionality

U - be Understood by all

M - Mandatory

A - Automated
```